app/src/main/res/values-fr/strings.xml

```diff
@@ -9,8 +9,8 @@
         <item name="valor">"Bravoure"</item>
         <item name="instinct">"Intuition"</item>
     </string-array>
-    <string name="place1">"Dans l\'ensemble, ton Pokémon&#8230;"</string>
-    <string name="place2">"Ses stats&#8230;"</string>
+    <string name="place1">Dans l\'ensemble, ton Pokémon&#8230;</string>
+    <string name="place2">Ses stats&#8230;</string>
```

Comment:
The quotation marks appeared on every single file including default (English). I don't think is part of syntax. \ is added so ' can appear right.